arwen/contexts/runtime.go

```diff
@@ -917,6 +920,21 @@ func (context *runtimeContext) MemStore(offset int32, data []byte) error {
 	return nil
 }
 
+func (context *runtimeContext) AddError(err error, otherInfo ...string) {
+	if err == nil {
+		return
+	}
+	if context.errors == nil {
+		context.errors = arwen.WrapError(err, otherInfo...)
+	} else {
+		context.errors = context.errors.WrapWithError(err, otherInfo...)
+	}
+}
+
+func (context *runtimeContext) GetAllErrors() error {
+	return context.errors
```

Comment:
Just a question: could we have concurrency issues here? Or all the times `GetAllErrors()` will be called only after all the errors will be added?

Reply:
yes, that should be the case, after an SC call